liam2/alignment.py

```diff
@@ -329,15 +366,17 @@ def compute(self, context, score, need, filter=None, take=None, leave=None,
 
         func = self.align_no_link if link is None else self.align_link
         return func(context, score, need, filter, take, leave, expressions,
-                    possible_values, errors, frac_need, link, secondary_axis)
+                    possible_values, errors, frac_need, link, secondary_axis, method = method)

```

Comment:
What is the policy ? The function has a lot of arguments and it took me some time to trace the error. Just asking though, you are the boss ;-)